framework-integrations/sagemaker/training/code/conf/dataset/uniref100.yaml

```diff
@@ -0,0 +1,8 @@
+train:
+  samples_before_next_set: null
+  paths: {uniref100: uniref/mila/uniref100_train.csv}
```

Comment:
Why is `uniref50: /opt/ml/input/data/train/uniref50_sample_100.csv` but `uniref100: uniref/mila/uniref100_train.csv`?

Reply:
Corrected to have only the uniref50 for sagemaker. Again you can change this to any data sources while training and can provide as a parameter. Please let me know if you need to have this default to uniref100